docs/index/mv-guide.md

```diff
@@ -217,40 +217,72 @@ release, user can do as following:
 Basically, user can manually trigger the operation by re-building the datamap.
 
 ## MV TimeSeries Support
-MV non-lazy datamap supports TimeSeries queries. Supported granularity strings are: year, month, week, day,
-hour,thirty_minute, fifteen_minute, ten_minute, five_minute, minute and second.
+Time series data are simply measurements or events that are tracked, monitored, downsampled, 
+and aggregated over time. Materialized views with automatic refresh mode supports TimeSeries queries.
 
- User can create MV datamap with timeseries queries like the below example:
+Carbondata provides built-in time-series udf with the below definition.
+
+```timeseries(time_series_column, 'granularity')```
+
+Columns provided in timeseries udf should be of TimeStamp/Date type.
+
+Below table describes the time hierarchy and levels that can be provided in a time-series udf, 
+so that it supports automatic roll-up in time dimension for query.
+
+| Granularity | Description |
+|---------------|----------------|
+| Year | Data will be aggregated over year |
```

Comment:
Is it Year or year? since in below example you are using "second"

Reply:
it is case insensitive